### ISSUE TITLE
Design public client profile activity

### ISSUE THREAD

**Issue by IagoSRL** [loconomics/loconomics#706]
As of #580 and #696 we replaced Profile activity with Listing. As commented at #705, this means we moved to a professional/listing focused activity, rather than the general Profile meant to display both professional and user/client public data.

This means **we lost an activity to display public client data**, and this issue exists to recover that, re-thinking it from scratch, starting with the design/prototyping phase and should continue with an implementation issue.

*BTW, I think this division in two activities is good since reduces code complexity and makes URLs propose more clear*.

**Comment by IagoSRL** [comment]
@joshdanielson I didn't find an issue for this, not sure if something was already designed or commented; maybe the name can be different to be clearer, let me know.

**Comment by joshdanielson** [comment]
@IagoSRL good catch. I'm wondering if we repurpose the /profile activity for users when they are a client (including service professionals) so that it's just the user's photo, bio, and reviews as it currently is for the client, but with the new styles.

**Comment by IagoSRL** [comment]
Yes, I hope is something like that, making clear is a client profile (even if the user being displayed is a professional, but that activity shows that one as a client, so reviews as a client and that; maybe a note about a client being a proffessional and linking to it's listing, but just link).
